extensions/figma-files/src/hooks/useFigmaApp.ts

```diff
@@ -0,0 +1,16 @@
+import { getApplications } from "@raycast/api";
+import { useState, useEffect } from "react";
+
+import type { Application } from "@raycast/api";
+
+export const useFigmaApp = (): Application | undefined => {
+  const [desktopApp, setDesktopApp] = useState<Application>();
+
+  useEffect(() => {
+    getApplications()
+      .then((apps) => apps.find((a) => a.bundleId === "com.figma.Desktop"))
+      .then(setDesktopApp);
+  }, []);
+
+  return desktopApp;
+};
```

Comment:
You probably can improve the hook and keep the detected desktop application. The desktop app doesn't change during the runtime of a command. So you could just load it once and then don't do it on every component mount.